fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/ApiJobStore.java

```diff
@@ -51,4 +53,21 @@
      * @return An Observable that emits a stream of all the JobRows in the store
      */
     Observable<JobRow> getAllRows();
+
+    /**
+     * This method takes a Set of JobRowFilters, ANDS them by default, and returns a cold observable that emits a
+     * stream of JobRows which satisfy the given filter.
+     * <p>
+     * Every field may not be filterable for every implementation of the {@code ApiJobStore} as the efficiency of
```

Comment:
'Any of the fields may be not' would be slightly clearer than 'Every field may not be'.  As written it seems to mean that sometimes all the fields are not filterable and sometimes they are all filterable.